resources/views/emails/participants/onEventConfirmation.blade.php

```diff
@@ -35,7 +40,7 @@
 	
 	<p>
 		BETALINGSINFORMATIE<br/>
-		Te betalen bedrag: € {{ $toPay }}<br/>
+		Te betalen bedrag: € {{ $payment->getTotalAmount() }}<br/>
 		Rekeningnummer: NL68 TRIO 0198 4197 83 t.n.v. Vereniging Anderwijs te Utrecht<br/>
 		Onder vermelding van: naam deelnemer + deze kampcode: {{ $event->code }}
 	</p>
```

Comment:
Als er geen betaling nodig is, zou ik de betalingsinfo ook niet laten zien, om verwarring te voorkomen.

Reply:
Deze staat al in de `@else` en daarmee wordt hij al niet getoond wanneer het gratis is.